examples/spouts/spout101/README.md

```diff
@@ -0,0 +1,181 @@
+to start the processing is short. 
+Therefore, a regular pipeline with a cron input
+that polls for new data might not be an optimal solution.
+
+A spout pipeline has three major differences from regular pipelines:
+- it runs continuously
+- it does not take an input repo. Instead, it consumes messages from a third-party messaging system.
+- `pfs/out` is not directly accessible. You will need to use `pachctl put file` to write into your output repo. In this example, we have used the `pachyderm-python` library. 
+
+
+You configure a spout in the 
+[pipeline specification](https://docs.pachyderm.com/latest/reference/pipeline_spec/)
+file by adding a `spout` section after its pipeline section.
+At a minimum, a spout pipeline will need a pipeline name,
+an empty spout section, and a transform section
+where you will specify the transformation you want
+to execute on the messages consumed.
+
+>![pach_logo](./img/pach_logo.svg) Note that the use
+of the [build pipelines](https://docs.pachyderm.com/latest/how-tos/developer-workflow/build-pipelines/) has not been made available
+in the 1.12.0 version of Pachyderm. Stay tuned.
+
+
+In this example, we have emulated the reception 
+of messages from a third-party messaging system
+to focus on the specificities of the spout pipeline. 
+![](./img/spout101.png)
+Feel free to explore more of our examples
+to discover how we used spout to listen
+for new S3 objects notifications via an Amazon™ SQS queue
+or connected to an IMAP email account
+to analyze the polarity of its emails.
+
+
+## Getting ready
+***Key concepts***
+- [Spout](https://docs.pachyderm.com/latest/concepts/pipeline-concepts/pipeline/spout/) concept.
+- [Spout](https://docs.pachyderm.com/latest/reference/pipeline_spec/#spout-optional) configuration 
+
+***Prerequisite***
+- A workspace on [Pachyderm Hub](https://docs.pachyderm.com/latest/pachhub/pachhub_getting_started/) (recommended) or Pachyderm running [locally](https://docs.pachyderm.com/latest/getting_started/local_installation/).
+- [pachctl command-line ](https://docs.pachyderm.com/latest/getting_started/local_installation/#install-pachctl) installed, and your context created (i.e., you are logged in)
+
+***Getting started***
+- Clone this repo.
+- Make sure Pachyderm is running. You should be able to connect to your Pachyderm cluster via the `pachctl` CLI. 
+Run a quick:
+```shell
+$ pachctl version
+
+COMPONENT           VERSION
+pachctl             1.12.0
+pachd               1.12.0
+```
+Ideally, have your pachctl and pachd versions match. At a minimum, you should always use the same major & minor versions of your pachctl and pachd. 
+
+## Example - Spout 101 
+***Goal***
+In this example,
+we will keep generating two random strings, 
+one of 1KB and one of 2KB,
+at intervals varying between 10s and 30s. 
+A spout pipeline will actively poll those data,
+then commit them to its output repo -
+in the form of a text file - using the **pachctl put file** command. 
```

Comment:
One thing to be a little careful of in terminology. `put file` is the API command. pachctl is technically the name of the CLI for the Pachdyerm API. So it doesn't 100% makes sense to say "use pachctl put file with the python client"... it probably more accurate to say "use the `put file` API call via the python client." I think JD or someone on the eng side could probably tell you what sounds best -- I'm not the best judge because I'm not a real developer.

Reply:
This comment applies to line 91 too